### PR TITLE
Create kunsh13.json

### PR DESCRIPTION
**Pull request by kunsh13**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

kunsh13.github.io/iptv

### CODE REVIEW

File: domains/kunsh13.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "kunsh13.is-a.dev",
+  "repo": "https://github.com/kunsh13/iptv",
+  "owner": {
+    "username": "kunsh13",
+    "email": "kunshmaurya@gmail.com"
+  },
+  "record": {
+    "CNAME": "kunsh13.github.io/iptv"
```
Comment (orxngc) [inline domains/kunsh13.json:9]:
```suggestion
    "CNAME": "kunsh13.github.io"
```
CNAMEs cannot contain slashes `/`.

Comment (kunsh13) [reply domains/kunsh13.json:9]: But I want a repo of mine to be hosted, how can I do that?

**Comment by kunsh13** [comment]
But I want a repo of mine to be hosted, how can I do that?

**Comment by orxngc** [comment]
> But I want a repo of mine to be hosted, how can I do that?

If your repo is `github.com/uwu/owo`, then using the CNAME `uwu.github.io` will allow you to use uwu.is-a.dev/owo to access that repository.

If you want to use uwu.is-a.dev without that extra /owo, then rename uwu/owo to uwu/uwu.github.io.

Now replace uwu and owo with your Github username and repository name, lol.

**Comment by orxngc** [comment]
I'm about to merge this; your site will be available at https://kunsh13.isa.dev/iptv.

**Comment by kunsh13** [comment]
> I'm about to merge this; your site will be available at https://kunsh13.isa.dev/iptv.





The site can't be reached

**Pull request merged**